cs/test/expressions/Clone.expressions

```diff
@@ -0,0 +1,817 @@
+.Lambda #Lambda1<System.Func`2[System.Object,System.Object]>(System.Object $obj) {
+    .Block(ExpressionsTest.Example $Example) {
+        .Block() {
+            $Example = .New ExpressionsTest.Example();
+            .Block(ExpressionsTest.Example $Example) {
+                $Example = (ExpressionsTest.Example)$obj;
```

Comment:
Because of the names we've given them, the source and destination both get called "Example" here. Let me see whether I can fix that easily.

Reply:
Done.